app/controllers/users.js

```diff
@@ -70,3 +78,36 @@ exports.listUsers = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.createAdminUser = (req, res, next) => {
+  const err = validateUserCreation(req.body);
+  if (err) return next(err);
+
+  userService
+    .getUserByEmail(req.body.email)
+    .then(user => {
+      if (user) {
+        if (user.firstName === req.body.firstName && user.lastName === req.body.lastName) {
+          user.permission = enums.PERMISSION.ADMINISTRATOR;
+          return user
+            .save()
+            .then(() => res.sendStatus(200))
+            .catch(error => {
+              throw errors.databaseError(error.message);
+            });
+        }
+
+        // Email is not available for the new user
+        throw errors.emailAlreadyInUse;
+      } else {
+        // If the email is available, a new administrator user is created
+        // with the body parameters
+        req.body.permission = enums.PERMISSION.ADMINISTRATOR;
```

Comment:
instead of doing this you could just do
```
return userService.createUser({ ...req.body, permission: enums.PERMISSION.ADMINISTRATOR }).then(newUser => {
```